front/app/components/ParticipationCTABars/VotingCTABar/CTAButton.tsx

```diff
@@ -125,33 +128,48 @@ const CTAButton = ({ phase, project }: Props) => {
   );
 
   return (
-    <Tippy
-      disabled={!disabledExplanation}
-      interactive={true}
-      placement="bottom"
-      content={disabledExplanation}
-    >
-      <Box width="100%">
-        <StyledButton
-          icon="vote-ballot"
-          buttonStyle="secondary"
-          iconColor={theme.colors.tenantText}
-          onClick={handleSubmitOnClick}
-          fontWeight="500"
-          bgColor={theme.colors.white}
-          textColor={theme.colors.tenantText}
-          id="e2e-voting-submit-button"
-          textHoverColor={theme.colors.black}
-          padding="6px 12px"
-          fontSize="14px"
-          disabled={!!disabledExplanation}
-          processing={processing}
-          className={disabledExplanation ? '' : 'pulse'}
-        >
-          <FormattedMessage {...messages.submit} />
-        </StyledButton>
-      </Box>
-    </Tippy>
+    <>
+      <Tippy
+        disabled={!disabledExplanation}
+        interactive={true}
+        placement="bottom"
+        content={disabledExplanation}
+      >
+        {/* We need to add a tabIndex when the explanation is shown to
+        make sure this is focusable to read the explanation */}
```

Comment:
```suggestion
        {/* We need to add a tabIndex of 0 to
        make sure this is keyboard-focusable so screen reader software can read the explanation */}
```
A little more context. :)